docs/architecture/adt-014-proportional-slashing.md

```diff
@@ -0,0 +1,87 @@
+# ADR 14: Proportional Slashing
+
+## Changelog
+
+- 2019-10-15: Initial draft
+
+## Context
+
+In Proof of Stake-based chains, centralization of consensus power amongst a small set of validators causes harm to the network due to increased risk of censorship, liveness failure, fork attacks, etc.  However, while this centralization causes a negative externality to the network, it is not directly felt by the stakers contributing towards delegating towards already large validators.  We would like a way to pass on the negative externality cost of centralization onto those large validators and their stakers.
+
+## Decision
+
+### Design
+
+To solve this problem, we will implement a procedure called Proportional Slashing.  The desire is that the larger a validator is, the more they should be slashed.  The first naive attempt is to make a validator's slash percent proportional to their share of consensus voting power.
+
+```
+slash_amount = power // power is the faulting validator's voting power.
+```
+
+However, this will incentivize validators with large amounts of stake to split up their voting power amongst accounts, so that if they fault, they all get slashed at a lower percent.  The solution to this is to take into account not just a validator's own voting percentage, but also the voting percentage of all the other validators who get slashed in a short time window.
```

Comment:
I think it's important to note here that "splitting" up accounts into Sybil validators isn't trivial unless the validator has a substantially large self-bond.

Reply:
Well, as a validator operator, I could just create two validators (i.e. `Sikka 1` and `Sikka 2`) and encourage my delegators to delegate to both.

Reply:
This is especially true for validators who delegation is primarily self-bonds.